fawkes/app_config/app_config.py

```diff
@@ -316,6 +324,8 @@ class AppConfig:
     """
 
     def __init__(self, config):
+        # Validate config with schema
+        self.validate_app_config_schema(config)
```

Comment:
The validation should be done after:
```python
config = self.inject_env_vars_as_values(config)
```
as there might be properties that we want to enforce later like length of a twitter api key etc. which are stored as secrets, but will be injected during run-time.

Reply:
Done